apps/ledger-live-mobile/src/families/celo/Delegations/Row.tsx

```diff
@@ -0,0 +1,127 @@
+import React from "react";
+import { View, StyleSheet, TouchableOpacity } from "react-native";
+import { useTranslation } from "react-i18next";
+import {
+  CosmosMappedDelegation,
+  CosmosMappedUnbonding,
+} from "@ledgerhq/live-common/families/cosmos/types";
+import { Currency } from "@ledgerhq/types-cryptoassets";
+import { useTheme } from "@react-navigation/native";
+import { Text } from "@ledgerhq/native-ui";
+import CounterValue from "../../../components/CounterValue";
+import ArrowRight from "../../../icons/ArrowRight";
+import LText from "../../../components/LText";
+import ValidatorImage from "../../cosmos/shared/ValidatorImage";
+import { LEDGER_BY_FIGMENT_VALIDATOR_GROUP_ADDRESS, revokableVotes, voteStatus } from "@ledgerhq/live-common/families/celo/logic";
+import { CeloAccount, CeloVote } from "@ledgerhq/live-common/lib/families/celo/types";
+import { formatAmount } from "./utils";
+
+type Props = {
+  account: CeloAccount,
+  vote: CeloVote;
+  currency: Currency;
+  onPress: (vote: CeloVote) => void;
+  isLast?: boolean;
+  getValidatorName: (vote: CeloVote) => string;
+};
+
+export default function DelegationRow({
+  account,
+  vote,
+  currency,
+  onPress,
+  isLast = false,
+  getValidatorName,
+}: Props) {
+  const { colors } = useTheme();
+  const { t } = useTranslation();
+  // const { validator, validatorAddress, formattedAmount, amount } = delegation;
+  const { validatorGroup, amount } = vote;
+  const validatorName = getValidatorName(vote) ?? "";
+  
+  // const actions = voteActions(vote);
+  const status = voteStatus(vote);
+
+  return (
+    <TouchableOpacity
+      style={[
+        styles.row,
+        styles.wrapper,
+        !isLast
+          ? { ...styles.borderBottom, borderBottomColor: colors.lightGrey }
+          : undefined,
+      ]}
+      onPress={() => onPress(vote)}
+    >
+      <View style={[styles.icon]}>
+        <ValidatorImage
+          size={42}
+          isLedger={validatorGroup === LEDGER_BY_FIGMENT_VALIDATOR_GROUP_ADDRESS}
```

Comment:
Nit and low LOGAF: consider using the `isDefaultValidatorGroupAddress` function here for consistency and not exporting `LEDGER_BY_FIGMENT_VALIDATOR_GROUP_ADDRESS`

Reply:
fixed